lib/src/nodes/config_node_path.cc

```diff
@@ -0,0 +1,17 @@
+#include <internal/nodes/config_node_path.hpp>
+
+using namespace std;
+
+namespace hocon {
+
+    config_node_path::config_node_path(path node_path, token_list tokens) :
+        _path(move(node_path)), _tokens(move(tokens)) { }
+
+    token_list config_node_path::get_tokens() const {
```

Comment:
We could stash the created list in config_node_include and return a reference to it. Depends on how the result of get_tokens is used; if we always want a copy to manipulate then stashing a copy in the object isn't great.

Let's leave it as-is, and we can optimize later if it's an issue.